src/client/core/transport.js

```diff
@@ -34,5 +28,24 @@ export function assertionFailed (err, callback) {
         err: err
     };
 
-    transport.asyncServiceMsg(assertionFailedMsg, callback);
+    asyncServiceMsg(assertionFailedMsg, callback);
+}
+
+// TODO: temporary solution before GH-50
+export function asyncServiceMsg (msg, callback) {
+    return transport.asyncServiceMsg.bind(transport)(msg)
```

Comment:
we can write just `transport.asyncServiceMsg(msg)` here. We were using `.bind` because it was assigned to the `asyncServiceMsg` variable.

Fix the same below